lib/s3encrypt.rb

```diff
@@ -126,6 +126,24 @@ def self.getfile(local_filename, remote_filename, bucket, app_context)
     fetch_file(s3client,keyvalue,local_filename,remote_filename,bucket)
   end
 
+  #########################################
+  # Helper method that bypasses writing a file to the system and returns a JSON object
+  # More for accessiing the data programatically so that a file does not have to be written and cleand up
+  #########################################
+  def self.getfile_as_json(remote_filename, bucket, app_context)
+    resp = getfile(nil, remote_filename, bucket, app_context)
+    return JSON.parse(resp.body.string)
```

Comment:
Do we need to require the json module for this?  It seems so but I don't see that in the pull request (which I already accepted)